src/sentry/search/events/filter.py

```diff
@@ -1129,3 +1131,17 @@ def _issue_filter_converter(self, search_filter: SearchFilter) -> Optional[Where
                 SearchValue(filter_values if search_filter.is_in_filter else filter_values[0]),
             )
         )
+
+    def _transaction_status_filter_converter(
+        self, search_filter: SearchFilter
+    ) -> Optional[WhereType]:
+        converted = _transaction_status_filter_converter(
+            search_filter, search_filter.key.name, self.params
+        )
+        return self._default_filter_converter(
```

Comment:
@getsentry/visibility Unsure if this is clean. I noticed a lot of the other converters will recursively call `convert_search_filter_to_condition` again, this didn't seem necessary for `transaction.status` as I'm not changing the key and I just want to get the string to int conversion for the status then do the default filter conversion.

Reply:
As long as you return a `WhereType` it should be fine so you could just construct a Condition directly here instead of calling `_default_filter_converter`

Reply:
👍  updated this to reimplement the filter conversion and directly return a Condition rather than using the default converter